templates/module/Source/module/Private/Generated/OLink/olinkclient.cpp.tpl

```diff
@@ -112,17 +132,17 @@ F{{$Iface}}{{Camel .Name}}ChangedDelegate& {{$Class}}::Get{{Camel .Name}}Changed
 {{$returnVal}} {{$Class}}::{{Camel .Name}}_Implementation({{ueParams "" .Params}})
 {
 	{{- if .Return.IsVoid }}
-	if (!m_node)
+	if (!m_sink->m_node)
 	{
-		UE_LOG(LogTemp, Warning, TEXT("%s has no node"), UTF8_TO_TCHAR(olinkObjectName().c_str()));
+		UE_LOG(LogTemp, Warning, TEXT("%s has no node"), UTF8_TO_TCHAR(m_sink->olinkObjectName().c_str()));
 		return;
 	}
 	InvokeReplyFunc Get{{$IfaceName}}StateFunc = [this](InvokeReplyArg arg) {};
-	m_node->invokeRemote("{{$ifaceId}}/{{.Name}}", { {{- ueVars "" .Params -}} }, Get{{$IfaceName}}StateFunc);
+	m_sink->m_node->invokeRemote("{{$ifaceId}}/{{.Name}}", { {{- ueVars "" .Params -}} }, Get{{$IfaceName}}StateFunc);
```

Comment:
use Name::createMemberId(olinkObjectName(), {{.Name}}) or keep the {{$ifaceId}} I mean mostly avoiding manually putting it together